core/src/main/scala/kafka/server/ForwardingManager.scala

```diff
@@ -17,25 +17,17 @@
 
 package kafka.server
 
-import kafka.metrics.KafkaMetricsGroup
+import java.nio.ByteBuffer
+
 import kafka.network.RequestChannel
+import kafka.utils.Logging
 import org.apache.kafka.clients.ClientResponse
-import org.apache.kafka.common.metrics.Metrics
 import org.apache.kafka.common.protocol.Errors
-import org.apache.kafka.common.requests.{AbstractRequest, AbstractResponse, EnvelopeRequest, EnvelopeResponse}
-import org.apache.kafka.common.utils.Time
+import org.apache.kafka.common.requests.{AbstractRequest, AbstractResponse, EnvelopeRequest, EnvelopeResponse, RequestHeader}
 
 import scala.compat.java8.OptionConverters._
 
-class ForwardingManager(metadataCache: kafka.server.MetadataCache,
-                        time: Time,
-                        metrics: Metrics,
-                        config: KafkaConfig,
-                        threadNamePrefix: Option[String] = None) extends
-  BrokerToControllerChannelManagerImpl(metadataCache, time, metrics,
-    config, "forwardingChannel", threadNamePrefix) with KafkaMetricsGroup {
-
-  private val forwardingMetricName = "NumRequestsForwardingToControllerPerSec"
+class ForwardingManager(channelManager: BrokerToControllerChannelManager) extends Logging {
```

Comment:
As ```ForwardingManager``` does not extend ```BrokerToControllerChannelManagerImpl```, the ```requestQueue``` can be declared as private now.

https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/server/BrokerToControllerChannelManagerImpl.scala#L58